### ISSUE TITLE
FAQ: include note about what happens to configs you had before installing OMZ

### ISSUE THREAD

**Issue by rrabinovitch** [ohmyzsh/ohmyzsh#9658]
Suggested text to be included in `readme` as a footnote in the installation section, or as an FAQ ("What happened to the configurations I had set up before installing Oh My Zsh?":
```
Note: If you had configurations saved in `.zshrc` prior to installing Oh My Zsh, those configurations are automatically  archived in a `.pre-oh-my-zsh` file and the new Oh My Zsh configurations will replace those previous configurations in `.zshrc`. You can import those archived configurations by adding `source ~/.pre-oh-my-zsh` at the end of your `.zshrc` file.
```

Context and research I've done related to this suggestion:
The only reference to the `.pre-oh-my-zsh` file I found in the OMZ documentation is in the context of uninstalling the framework, and none of the obvious google searches addressed resulted in answers to "what happened to my previous configs?" - having this noted in the context of the installation instructions or FAQs would have saved me a bit of confusion and surprise thinking I'd lost those configs.

I'm down to submit a PR for this as a `readme` edit, but wasn't sure if it's preferable for this note to live in the wiki docs.

**Comment by mcornella** [comment]
> You can import those archived configurations by adding `source ~/.pre-oh-my-zsh` at the end of your `.zshrc` file.

This is wrong. All settings in the previous zshrc file should be ported to the new one. There are two issues here:

- We shouldn't move existing zshrc files but provide a snippet which the user should add on the existing zshrc file.
- When we uninstall we should take that into account and only delete OMZ related lines in the zshrc file instead of renaming it altogether.

**Comment by rrabinovitch** [comment]
When I installed omz, it did not port or copy any of my previous configs into the updated zshrc. They were simply replaced by the new omz configs.

**Comment by mcornella** [comment]
Yeah sorry, I meant that the user should copy them instead of sourcing the `.pre-oh-my-zsh` file.

**Comment by mcornella** [comment]
Fixed in #10315.
